cmd/ebrelayer/main.go

```diff
@@ -155,18 +155,27 @@ func RunInitRelayerCmd(cmd *cobra.Command, args []string) error {
 	validatorMoniker := args[3]
 	mnemonic := args[4]
 
-	// Universal logger
-	logger := tmLog.NewTMLogger(tmLog.NewSyncWriter(os.Stdout))
+	logger, err := zap.NewProduction()
```

Comment:
Is this the configuration we want?  It enables stack traces and sampling, and I suspect we don't want either of those in production.

Reply:
FYI, log messages with stack traces look like this, so maybe it's not so bad:

`{"level":"error","ts":1614707042.5673304,"caller":"relayer/ethereum.go:212","msg":"failed to get the last ethereum block from level db.","error message":"leveldb: not found","stacktrace":"github.com/Sifchain/sifnode/cmd/ebrelayer/relayer.EthereumSub.Start\n\t/home/james/workspace/prsifnode/cmd/ebrelayer/relayer/ethereum.go:212"}
`

Reply:
yes, if there is an error in process. the stacktrace can help us for debug.

Reply:
We need to make sure sampling is turned off though - we want all logs.